### PR TITLE
[TypeScript][Skill] Add Functional Tests

### PR DESCRIPTION
**Pull request by LCarbonellRod**
Related to 3290 

### Purpose
*What is the context of this pull request? Why is it being done?*
There is no functional test applied for Skill in TypeScript.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
- Copy and adapt C# Skill functional test as the responses' length between C# and TS are different
- Add yml file for the TypeScript Skill

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
\-

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
\-

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [X] I have added or updated the appropriate tests	
- [X] I have updated related documentation

### CODE REVIEW

File: templates/typescript/samples/sample-skill/functional-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+﻿# Functional Tests for TypeScript Skill
+Follow these [steps](https://microsoft.github.io/botframework-solutions/solution-accelerators/tutorials/enable-continuous-integration/typescript/3-configure-build-steps/) to configure the functional tests using the `sample-assistant.yml`.
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/functional-tests/README.md:2]:
```suggestion
Follow these [steps](https://microsoft.github.io/botframework-solutions/solution-accelerators/tutorials/enable-continuous-integration/typescript/3-configure-build-steps/) to configure the functional tests using the `sample-skill.yml`.
```

File: templates/typescript/samples/sample-skill/functional-tests/sample-skill.yml [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+    
+     Write-Host "Dispatch CLI"
+     Write-Host "===="
+     dispatch -v
+    
+     Write-Host "BotSkills CLI"
+     Write-Host "===="
+     botskills -v
+    
+     Write-Host "BF SDK and Solutions package versions being used by the VA Sample for this nightly build"
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/functional-tests/sample-skill.yml:94]:
```suggestion
     Write-Host "BF SDK and Solutions package versions being used by the Skill Sample for this nightly build"
```

File: templates/typescript/samples/sample-skill/functional-tests/SkillSample.FunctionalTests/DirectLineClientTests.cs [hunk 3]
```diff
@@ -225,10 +214,10 @@ private void GetEnvironmentVars()
         {
             if (string.IsNullOrWhiteSpace(_directLineSecret) || string.IsNullOrWhiteSpace(_botId))
             {
-                _directLineSecret = Environment.GetEnvironmentVariable("DIRECTLINEKEY");
+                _directLineSecret = Environment.GetEnvironmentVariable("DIRECTLINE");
                 if (string.IsNullOrWhiteSpace(_directLineSecret))
                 {
-                    Assert.Inconclusive("Environment variable 'DIRECTLINEKEY' not found.");
+                    Assert.Inconclusive("Environment variable 'DIRECTLINE' not found.");
```
Comment (Batta32) [inline templates/typescript/samples/sample-skill/functional-tests/SkillSample.FunctionalTests/DirectLineClientTests.cs]: Is this okay?

Comment (LCarbonellRod) [reply templates/typescript/samples/sample-skill/functional-tests/SkillSample.FunctionalTests/DirectLineClientTests.cs]: Yes! The C# Skill [changes](https://github.com/southworks/botframework-solutions/pull/375/files#diff-393da35a03e7e3bd744aa8f94ac216a3R180) has it.

Comment (Batta32) [reply templates/typescript/samples/sample-skill/functional-tests/SkillSample.FunctionalTests/DirectLineClientTests.cs]: Ack, following the yml is okay.

**Comment by LCarbonellRod** [comment]
> Are we using the `SampleActionInput` file?

No, we're not. I removed it.

**Pull request closed without merging**